pkg/api/feed.go

```diff
@@ -64,6 +64,14 @@ func (s *Service) feedGetHandler(w http.ResponseWriter, r *http.Request) {
 		queries.At = time.Now().Unix()
 	}
 
+	headers := struct {
```

Comment:
Should we keep the queries above with `At` and `After` ? Also bellow there is a lookup using this queries.

Reply:
it may be needed for epoch based feeds.
I created an issue about that #4830